libs/MVS/SceneTexture.cpp

```diff
@@ -509,8 +530,35 @@ bool MeshTexture::ListCameraFaces(FaceDataViewArr& facesDatas, float fOutlierThr
 		faceMap.create(imageData.height, imageData.width);
 		depthMap.create(imageData.height, imageData.width);
 		RasterMesh rasterer(vertices, imageData.camera, depthMap, faceMap);
+		//Creating mask for the image border
+		rasterer.mask = cv::Mat::zeros(imageData.height, imageData.width, CV_8U);
+		cv::Mat imageCopy = imageData.image.clone();
```

Comment:
interesting implementation, but pls try using cv::floodFill() as discussed, which is designed and optimized to do exactly this, plus the code will become easier to read